.github/workflows/checks.yml

```diff
@@ -0,0 +1,89 @@
+name: PR checks
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+    paths-ignore:
+      - "LICENSE"
+      - "**/.gitignore"
+      - "**.md"
+      - "**.adoc"
+      - "*.txt"
+      - "docsimg/**"
+      - ".ci/jenkins/**"
+
+env:
+  GO_VERSION: 1.19
+
+jobs:
+  unit-tests:
+    concurrency:
+      group: kogito-serverless-operator-unit-tests-${{ github.head_ref }}
+      cancel-in-progress: true
+    timeout-minutes: 60
+    name: Unit tests And Coverage
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
@radtriste can we add this packages installed for podman in the Container-builder ?  https://github.com/kiegroup/container-builder/blob/main/.github/workflows/test-go.yml#L13..L17

Reply:
Is that mandatory ?
I mean there is no issue here